### ISSUE TITLE
Accuracy training

### ISSUE THREAD

**Issue by AWuhrmann** [csho33/bacteria-ID#5]
Dear @csho33,
Thank you for your excellent work. It seems like the accuracy on the training set (not finetuning) goes up very quickly when training, reaching already 97+% val acc. after one epoch. I couldn't find any information regarding that in the paper nor in the repo. Is it normal ? 

Thank you in advance,

Arthur W.
